ocaml/typing/typedtree.mli

```diff
@@ -948,8 +948,15 @@ and constructor_declaration =
      cd_attributes: attributes;
     }
 
+and constructor_argument =
+  {
+    ca_global: Mode.Global_flag.t loc;
```

Comment:
Mind commenting that `ca_global`'s loc field is `Location.none` when `txt` is Unrestricted?